frontend/src/partners/Partner.js

```diff
@@ -11,15 +11,28 @@ export default function Partner() {
     partnerData,
   } = usePartnerData()
 
-  let isCreate = window.location.pathname === '/partners/new'
-  return (
+  return window.location.pathname === '/partners/new' ? (
```

Comment:
Na, miért?

Reply:
Szia!
Az előző változat nem töltötte ki a módosítás form alatt az adatokat (kilogoltam őket, úgy ott voltak, de nem volt kitöltve a szerkesztendő partnernél semmi). Az employee formnál ez működött, így azt másoltam.

Reply:
Ha valami nem működik akkor javítsuk, ne elfedjük a hibát. Azért nem tölti ki, mert lehet még nincs partner adat, amikor az initialValues-t megkapja a Formik, ha utána meg változik az initialValues, akkor nem tölti már ki. Ez a megoldás azért működik, mert a partnerData-ra van ellenőrizve renderelés előtt a szerkesztés esetén, nem azért, mert copy-paste-elt duplicated code.

Reply:
Ez a rész megvolt az elején is, csak nem tudjuk megoldani. Lehet h tök egyszerű lenne, de nem megy :(